### ISSUE TITLE
Home page color contrast and alt text

### ISSUE THREAD

**Issue by pdpinch** [mitodl/ocw-hugo-themes#413]
#### Acceptance Criteria:

- [ ] Orange text on white background doesn't have enough contrast. Remove the span tag so all the text is black 
   See <img width="159" alt="image" src="https://user-images.githubusercontent.com/430126/152714747-172a104b-a1ea-4e13-b0f6-9f35eaff9a1f.png">
- [ ] Use the [wcag-color-contrast-check](https://chrome.google.com/webstore/detail/wcag-color-contrast-check/plnahcmalebffmaghcpcmpaciebdhgdf) chrome extension or similar to review any other contrast issues 
- [ ] Set `alt=""` for course thumbnails (we could extract the alt text from the course site, but these images are essentially decorative on the home page and the screen reader user would prefer to link to the course quickly
- [ ] add `aria-hidden="true" tabindex="-1"` to the `a` enclosing the image to hide them from screen readers 
- [ ] review the rest of the images on the page if they also need to be hidden. 

#### Related Issues

- page landmarks review 
- homepage carousel review
